compiler/rustc_mir_build/src/thir/pattern/deconstruct_pat.rs

```diff
@@ -1144,6 +1150,7 @@ impl<'p, 'tcx> Fields<'p, 'tcx> {
         })
     }
 
+    // tidy-ticket-wildcards
```

Comment:
I would have expected that we reuse the tag for all the places that need to be in sync. E.g. if I modify `wildcards()` here and not `arity()`, it would ping me "you should also check `arity`".

Reply:
Added grouping for tags, should write
```
tidy error: hash for tag `tidy-ticket-ast-from_token` in path `compiler/rustc_ast/src/token.rs` mismatch:
  actual: `eecacc0fe4c615f7c71409c3ebfcff80`, expected: `70666de80ab0194a67524deeda3c01b8`
  Verify that tags `["tidy-ticket-ast-from_token", "tidy-ticket-ast-can_begin_literal_maybe_minus", "tidy-ticket-rustc_p
arse-can_begin_literal_maybe_minus"]` in sync.
```

Reply:
Nice! Could you make it explain what it's about for those who don't know? Something like: "the code block tagged with `tidy-ticket-ast-from_token` has changed. It's likely that code blocks with the following tags need to be changed too. Once that is done, run `tidy` again with the `--blessed` flag to update the hashes. The related tags are  `["tidy-ticket-ast-from_token", "tidy-ticket-ast-can_begin_literal_maybe_minus", "tidy-ticket-rustc_parse-can_begin_literal_maybe_minus"]`"

Reply:
Added `tidy-keep-sync-with=` and updated error message with parts from your suggestion.